components/event-bus/internal/knative/util/kn-eventing.go

```diff
@@ -55,10 +55,31 @@ if err := k.CreateSubscription("my-sub", namespace, channelName, &uri); err != n
 return
 */
 
+// Use it cause it should be mocked
+type KnativeLibIntf interface {
```

Comment:
Please change name of the `KnativeLibIntf` to something else which does not include the `Intf` suffix.

Reply:
Done